recipes/tokyocabinet/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+./configure --prefix=$PREFIX \
+	    --with-zlib=$PREFIX \
+	    --with-bzip=$PREFIX
+
+make
+if [[ $(uname) == Linux ]]; then
+    make check
```

Comment:
I had some trouble on mac at first and forgot to remove this after testing. Sorry.